docs/user-guide/disassembler-backends/binary_ninja.md

```diff
@@ -2,44 +2,60 @@
+    You need to have Binary Ninja installed along with a valid **commercial** Binary Ninja license to run OFRAK Binary Ninja natively. 
 
-```bash
-python3 \
-  -c 'import json, sys; print(json.load(sys.stdin)[0]["serial"])' \
-  < license.dat \
-  > serial.txt
-```
+    1. Create a virtual environment to which you will install code:
+        ```
+        % python3 -m venv venv
+        % source venv/bin/activate
+        ```
+    1. Install `ofrak` and its dependencies.
+    1. Run `make {install, develop}` inside of the ['ofrak_binary_ninja/'](https://github.com/redballoonsecurity/ofrak/tree/master/disassemblers/ofrak_binary_ninja) directory to install OFRAK Binary Ninja.
+    1. Next, install the Binary Ninja Python APIs in your virtual environment
+        ```python
+        % python3 "/Applications/Binary Ninja.app/Contents/Resources/scripts/install_api.py" -v
+        ```
 
-The command `python3 build_image.py --config ofrak-binary-ninja.yml --base --finish` will build an image using Docker BuildKit secrets so that neither the license nor serial number are exposed in the built Docker image. (If [Docker BuildKit](https://docs.docker.com/develop/develop-images/build_enhancements/) is not enabled in your environment, precede the `python3 build_image.py` command with `DOCKER_BUILDKIT=1`.)
+=== "Docker"
 
-The Docker container should be run with the same license file from the installation step. The license can then be mounted into the Docker container at location `/root/.binaryninja/license.dat` by adding the following arguments to the `docker run` command:
+    You need to have a valid **headless** Binary Ninja license to build and run the Docker image. [Read about the environment setup](../../environment-setup.md#binary-ninja) for more details.
 
-```
---mount type=bind,source="$(pwd)"/license.dat,target=/root/.binaryninja/license.dat 
-```
+    To build the image, the license should be placed in the project's root directory and named `license.dat`. The serial number needs to be extracted from that file into a file named `serial.txt`. This can be done with the following command:
 
-For example:
+    ```bash
+    python3 \
+      -c 'import json, sys; print(json.load(sys.stdin)[0]["serial"])' \
+      < license.dat \
+      > serial.txt
+    ```
 
-```bash
-# This simple command...
-docker run -it redballoonsecurity/ofrak/binary-ninja bash
+    The command `python3 build_image.py --config ofrak-binary-ninja.yml --base --finish` will build an image using Docker BuildKit secrets so that neither the license nor serial number are exposed in the built Docker image. (If [Docker BuildKit](https://docs.docker.com/develop/develop-images/build_enhancements/) is not enabled in your environment, precede the `python3 build_image.py` command with `DOCKER_BUILDKIT=1`.)
```

Comment:
Consider rephrasing to make it more obvious that `DOCKER_BUILDKIT=1` should be at the beginning of the command.

I can imagine a situation where a user does not read closely, and misses that they need it, causing the build to fial for them. I wonder if it makes sense to always set it equal to 1 in the `env` when we run the `subprocess` command in `build_image.py`? Thoughts @whyitfor @EdwardLarson ?